src/engine/globals.cpp

```diff
@@ -399,6 +399,7 @@ void initialize() {
     rootPropertyOwner->addPropertySubOwner(global::dashboard);
 
     rootPropertyOwner->addPropertySubOwner(global::userPropertyOwner);
+    rootPropertyOwner->addPropertySubOwner(global::openSpaceEngine);
```

Comment:
This seems to affect the order in which they are added in the Settings menu as well.

I suggest we put it a bit higher up in the list, maybe next to `RenderEngine`? Would make more sense, conceptually, at least to me

Reply:
@alexanderbock ?

Reply:
Yea, I agree with @WeirdRubberDuck    in addition it might be worth thinking about sorting the Settings list alphabetically additionally?